src/core/serializedtypes.js

```diff
@@ -258,8 +258,8 @@ function parse(so) {
 
   const field_bits = tag_byte & 0x0f;
   let field_name = (field_bits === 0)
-    ? field_name = binformat.fields[type_bits][so.read(1)[0]]
```

Comment:
yes, I think line 261&262 get executed before declaration at 260 so field_name could get implicitly declared as global? anyway simpler is better